package/network/utils/iproute2/patches/170-ip_tiny.patch

```diff
@@ -25,19 +25,20 @@
  		sed -n '/'$$s'[^ ]* =/{s:.* \([^ ]*'$$s'[^ ]*\) .*:extern char \1[] __attribute__((weak)); if (!strcmp(sym, "\1")) return \1;:;p}' $$files ; \
 --- a/ip/ip.c
 +++ b/ip/ip.c
-@@ -64,10 +64,16 @@ static void usage(void)
+@@ -64,11 +64,17 @@ static void usage(void)
  	fprintf(stderr,
  		"Usage: ip [ OPTIONS ] OBJECT { COMMAND | help }\n"
  		"       ip [ -force ] -batch filename\n"
 +#ifndef IPROUTE2_TINY
- 		"where  OBJECT := { link | address | addrlabel | route | rule | neigh | ntable |\n"
- 		"                   tunnel | tuntap | maddress | mroute | mrule | monitor | xfrm |\n"
- 		"                   netns | l2tp | fou | macsec | tcp_metrics | token | netconf | ila |\n"
- 		"                   vrf | sr | nexthop | mptcp }\n"
+		"where  OBJECT := { address | addrlabel | fou | help | ila | l2tp | link |\n"
+		"                   macsec | maddress | monitor | mptcp | mroute | mrule |\n"
+		"                   neighbor | neighbour | netconf | netns | nexthop | ntable |\n"
+		"                   ntbl | route | rule | sr | tap | tcpmetrics |\n"
+		"                   token | tunnel | tuntap | vrf | xfrm }\n"
 +#else
-+		"where  OBJECT := { link | address | route | rule | neigh | tunnel | maddress |\n"
-+		"                   mroute | mrule | monitor | netns | macsec | token | ila |\n"
-+		"                   vrf | sr }\n"
++		"where  OBJECT := { address | ila | route | rule | link | neigh | tunnel |\n"
```

Comment:
it looks like upstream changed the alphabetical ordering, could you do the same change for the tiny version.